docs/data/routes/guides/multisite-guide/implementation-patterns/en.md

```diff
@@ -0,0 +1,64 @@
+---
+name: implementation-patterns
+routeTemplate: ./data/component-templates/guide.yml
+title: Implementation Patterns
+---
+
+## Multisite in JSS
+With the out-of-the-box configuration, each JSS app creates one site definition and generates an independent code bundle. So to support multisite scenarios, it is necessary to create and deploy multiple JSS apps.
+
+If you deploy 3 JSS apps to the same Sitecore instance, this will create 3 independent sites, and 3 independent webpack bundles on the filesystem that would be structured like this:
```

Comment:
Maybe it is good to mention that even two sites use the same jss app, each of them have to have the own folder with according name under the dist folder with the same jss app code.